src/sentry/integrations/pagerduty/notify_action.py

```diff
@@ -74,7 +73,7 @@ def _validate_service(self, service_id: int, integration_id: int | None) -> None
                 params=params,
             )
 
-    def clean(self) -> Mapping[str, Any]:
+    def clean(self) -> Dict[str, Any]:
```

Comment:
This is more of a personal preference thing, but you can annotate this with `dict[str, Any]` and not have to import the typing class.

Reply:
I agree, I'll make a change.

Reply:
Ugh I'm getting an error
```
Subscript for class "dict" will generate runtime exception; enclose type annotation in quotes
```
but it goes away when I do `Dict`.

Reply:
Weird. OK.